translations/translated/mudlet_en_US.ts

```diff
@@ -1,10 +1,29 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!DOCTYPE TS>
 <TS version="2.1" language="en_US" sourcelanguage="en">
+<context>
+    <name>GLWidget</name>
+    <message numerus="yes">
+        <location filename="../../src/glwidget.cpp" line="383"/>
+        <source>You have a map loaded (%n room(s)), but Mudlet does not know where you are at the moment.</source>
+        <translation>
+            <numerusform>You have a map loaded (one room), but Mudlet does not know where you are at the moment.</numerusform>
```

Comment:
:thinking: The singular case is a bit of a corner case - but one a new user will likely hit if they are starting their own map. Mudlet is basically announcing there: "hey there is a single room in this map" but then says "I can not work out which room you are in at the moment" - technically we *could* force the player room to be that single one but is that too much interference - "*Mudlet knows best*"?

Reply:
I am happy with the PR as it is now - this UI point can easily be deferred to another PR...

Reply:
That's a good point, we might as well pre-set it to that room!

Reply:
To be honest had same thing. But this rather mean that map is not really complete... And this single room, might be not correct one :)

Reply:
As a future improvement we might actually try to add list of let's say 3 random rooms or something. But the problem is that we can centerview room only, same with single room. Mapping script either way will have to take care of rest.

Reply:
I'm not sure what do you mean, because if there's 1 room in the map, it can't be wrong to select it?

Reply:
We're going deeper into corner cases, it might be in area Zone A, while player is in Zone B :)

Reply:
Yeah sure, it could very well be an incorrect one, but the mapper being incorrect can happen on load. If your mapper script is working right (we can build assuming that), it'll get correct fast with a `look` or a move.

Reply:
But if there are no rooms (left) in Zone B can the player be truly ***in*** that zone.:laughing: 

Is this what they mean when they say something is a [pathological case](https://en.wikipedia.org/wiki/Pathological_(mathematics)#Computer_science)...